src/components/Tag/Tag-story.js

```diff
@@ -6,16 +6,53 @@ import { Tag, TagWrapper } from '../../index';
 import { withReadme } from 'storybook-readme';
 import readme from './README.md';
 
+class TagUpdate extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      action: 'emoji',
+      count: 14,
+    };
+  }
+
+  tagClick = () => {
+    this.setState({
+      action: 'success',
+      count: 15,
+    });
+  };
+
+  render() {
+    return (
+      <LightWrapper>
+        <TagWrapper aria-label="Tags">
+          <Tag
+            action={this.state.action}
+            actionFunction={this.tagClick}
+            actionDescription="Add a point"
+          >
+            Offering Managment &middot; {this.state.count}
+          </Tag>
+        </TagWrapper>
+      </LightWrapper>
+    );
+  }
+}
+
 const actionTagProps = {
-  leftAction: true,
```

Comment:
@stpCollabr8nLstn Removed this as I couldn't find its usage - that OK?

Reply:
👍